easybuild/easyconfigs/g/GLM/GLM-0.9.8.3-GCCcore-5.4.0.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'CMakeMake'
+
+name = 'GLM'
+version = '0.9.8.3'
+
+homepage = 'https://github.com/g-truc/glm'
+description = """OpenGL Mathematics (GLM) is a header only C++ mathematics library for graphics software based on
+ the OpenGL Shading Language (GLSL) specifications."""
+
+toolchain = {'name': 'GCCcore', 'version': '5.4.0'}
```

Comment:
Hmm, wouldn't it make sense to make this `dummy`, since it's headers only? Is anything actually being compiled?

Also, why, WHY, do they use `cmake` to install header files?!